Formula/g/glib.rb

```diff
@@ -6,9 +6,9 @@ class Glib < Formula
   url "https://download.gnome.org/sources/glib/2.84/glib-2.84.0.tar.xz"
   sha256 "f8823600cb85425e2815cfad82ea20fdaa538482ab74e7293d58b3f64a5aff6a"
   license "LGPL-2.1-or-later"
+  revision 1
 
   bottle do
-    rebuild 1
```

Comment:
Please don't touch bottle blocks

Reply:
Do I need to remove revision then? I am not sure.

Reply:
Keep `revision 1`, do not touch `bottle` block.

Reply:
OK that is a good lesson, thank you!